pkg/model/scalewaymodel/instances.go

```diff
@@ -75,6 +78,15 @@ func (b *InstanceModelBuilder) Build(c *fi.CloudupModelBuilderContext) error {
 			instance.Role = fi.PtrTo(scaleway.TagRoleWorker)
 		}
 
+		// If the instance's commercial type is one that has no local storage, we have to specify for the
+		// block storage volume a big enough size (default size is 10GB)
+		for _, commercialType := range commercialTypesWithBlockStorageOnly {
+			if strings.HasPrefix(ig.Spec.MachineType, commercialType) {
+				instance.VolumeSize = fi.PtrTo(50)
```

Comment:
Could you put the default size in a const? Also are you sure that this is the correct way to specify a size ? I thought it was: 		volumeSize    = 20 * scw.GB for instance

Reply:
I put the value in a constant, but for the second remark, in the model part I have to store the value as an int and not a Scaleway-specific type. That's why I only put the size in GB so that it doesn't overflow. 
When I actually make the request to create the instance ([here](https://github.com/kubernetes/kops/blob/9816fd941624f73e5b2a819bce58f443d703042e/upup/pkg/fi/cloudup/scalewaytasks/instance.go#L244)) , I retrieve the value and multiply it by size.GB.
NB: I noticed that `50` was also hardcoded in that file, I'll use the constant instead.